pkg/backingstore/reconciler.go

```diff
@@ -1160,6 +1167,16 @@ func (r *Reconciler) needUpdate(pod *corev1.Pod) bool {
 			return true
 		}
 	}
+
+	var noobaaLogEnv = "NOOBAA_LOG_LEVEL"
+	var configMapLogLevel = r.CoreAppConfig.Data[noobaaLogEnv]
+	noobaaLogEnvVar := util.GetEnvVariable(&c.Env, noobaaLogEnv)
+
+	if (configMapLogLevel != noobaaLogEnvVar.Value) {
+		r.Logger.Warnf("NOOBAA_LOG_LEVEL Env variable change detected: on the config map")
```

Comment:
you missed the values in the print:
```suggestion
		r.Logger.Warnf("NOOBAA_LOG_LEVEL Env variable change detected:(%v) on the config map(%v)", noobaaLogEnvVar.Value, configMapLogLevel)
```